---
layout: default
---

I am a formally trained biochemist with graduate and post-doctoral work in Computational Systems Biology. Most of my work has been on applications of machine learning tools for the inference of biologically relevant networks, with a particular focus on gene regulatory networks. You can find my CV [here](https://github.com/diogocamacho/CV/blob/master/DMC.pdf).

I currently lead the [Systems Discovery]() group at the [Wyss Institute for Biologically Inspired Engineering](wyss.harvard.edu). Do not hesitate to [reach out](mailto:diogo.camacho@wyss.harvard.edu) to find out about what we are currently working on, new job opportunities, or engage in novel collaborations.



## GitHub repos
A list of my public repos:

 - [DrugMineR](https://github.com/diogocamacho/drugminer): An R package for querying drugs/compounds in order to extract metadata on these drugs. Data is collated from PubChem and KEGG for now, with expansions coming in the future.

 - [nsmblR](http://www.github.com/diogocamacho/nsmblR): An R package for the inference of ensemble model of gene regulatory networks. For this ensemble I use 7 different inference algorithms that focus on different statistical and mathematical methodologies.

 - [DiffNet](http://www.github.com/diogocamacho/diffnet): An R package for the identification of topological differences in gene regulatory networks based on a compendium of expression data.  

 - [rpegeos](http://www.github.com/diogocamacho/rpegeos): An R package for the enrichment of pathways using natural language processing concepts.

 - [oranges](http://www.github.com/diogocamacho/oranges): An R package for the enrichment of pathways using Fisher's exact test. Limited to human pathways.

 - [Jalili et al., 2019](http://www.github.com/diogocamacho/jalili_microbiome): Analyses of microbiome data in a microfluidic anaerobic chamber. Paper by Jalili et al. [paper](https://www.nature.com/articles/s41551-019-0397-0)

 - [Tovaglieri et al., 2010](http://www.github.com/diogocamacho/tovaglieri_metabolites): Computational analyses of metabolomics and 16S data from Organ-Chip samples exposed to enterohemorraghic E. coli. [paper](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-019-0650-5)
