---
layout: default
---

Formally trained biochemist with graduate and post-doctoral work in Computational Systems Biology. Most of my work has been on applications of machine learning tools for the inference of biologically relevant networks, with a particular focus on gene regulatory networks. 

## GitHub repos
A list of my public repos:

 - [nsmblR](http://www.github.com/diogocamacho/nsmblR): An R package for the inference of ensemble model of gene regulatory networks. For this ensemble I use 7 different inference algorithms that focus on different statistical and mathematical methodologies.

 - [DiffNet](http://www.github.com/diogocamacho/diffnet): An R package for the identification of topological differences in gene regulatory networks based on a compendium of expression data.  

 - [rpegeos](http://www.github.com/diogocamacho/rpegeos): An R package for the enrichment of pathways using natural language processing concepts.

 - [oranges](http://www.github.com/diogocamacho/oranges): An R package for the enrichment of pathways ysing Fisher's exact test. Limited to human pathways.

 - [Jalili et al., 2019](http://www.github.com/diogocamacho/jalili_microbiome): Analyses of microbiome data in a microfluidic anaerobic chamber. Paper by Jalili et al. (in press)

 - [Tovaglieri et al., 2010](http://www.github.com/diogocamacho/tovaglieri_metabolites): Computational analyses of metabolomics and 16S data from Organ-Chip samples exposed to enterohemorraghic E. coli. [paper](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-019-0650-5)

